src/main/java/htsjdk/samtools/SAMUtils.java

```diff
@@ -214,17 +218,19 @@ private static byte charToCompressedBaseLow(final int base) {
             case 'b':
                 return COMPRESSED_B_LOW;
             default:
-                throw new IllegalArgumentException("Bad  byte passed to charToCompressedBase: " + base);
+                throw new IllegalArgumentException("Bad base passed to charToCompressedBaseLow: " + Character.toString((char)base) + " in read: " + readName);
```

Comment:
good idea to output the character equivalent of the byte number, but you probably also want the byte value itself in case its a non-printing or formatting character